doc/introduction/configuration.rst

```diff
@@ -52,4 +52,35 @@ Configuration options
 
 **port (int)** (*optional*)
     The port to be used when connecting to the remote service.
-    Corresponding environment variable: ``SF_API_PORT``
\ No newline at end of file
+    Corresponding environment variable: ``SF_API_PORT``
+
+Store your account
+------------------
+
+Using the :func:`~.store_account` function, a configuration file can be created easily. It only requires specifying the authentication token. Apart from that, further configuration options can be passed as keyword arguments.
+
+Configure for the current SF project
+************************************
+
+The following is an example for using ``store_account`` with defaults:
+
+.. code-block:: python
+
+    import strawberryfields as sf
+    sf.store_account("MyToken")
+
+where ``"MyToken"`` contains the user specific authentication token.
+
+It is advised to execute this code snippet **only once** per configuration. This should be done in the same directory where the SF project can be found, separately from executing any other Strawberry Fields scripts. Using the default options it will store the account in the *current working directory* by creating a ``config.toml`` file.
```

Comment:
What is "the SF project"? :thinking:

Reply:
It was meant to refer to Strawberry Fields project, but I'll rephrase, thanks! :)